packages/accordions/src/elements/accordion/components/Panel.tsx

```diff
@@ -5,15 +5,17 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { useCallback, forwardRef, HTMLAttributes } from 'react';
+import React, { useCallback, useRef, forwardRef, HTMLAttributes } from 'react';
 import debounce from 'lodash.debounce';
-import { useCombinedRefs } from '@zendeskgarden/container-utilities';
-import { useAccordionContext, useSectionContext } from '../../../utils';
+import mergeRefs from 'react-merge-refs';
+import { useAccordionContext, useSectionContext, IAccordionContext } from '../../../utils';
 import { StyledPanel, StyledInnerPanel } from '../../../styled';
 
+type PanelProps = IAccordionContext | { isBare?: boolean; isExpanded?: boolean };
```

Comment:
Not a blocker, but I think the TypeScript compiler will pick up the `isBare` type from the `IAccordionContext` interface.